third_party/OpenAPI/example.swagger.json

```diff
@@ -0,0 +1,234 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "example.proto",
```

Comment:
I think you might want to change that, as well as a file name :)

Reply:
I think you can also convert it to YAML for easier comparison with manually-written spec.

Reply:
https://github.com/Percona-Lab/percona-version-service/pull/1/commits/e06c492963205bacbee0f3283550cd01897c56ec
https://github.com/Percona-Lab/percona-version-service/pull/1/commits/060b2a00049205ee38570d29610f9807bed76fef